scripts/check_delta.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/python3
+
+import argparse
+import os
+import sys
+import xml.etree.ElementTree as ET
+import shutil
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+            description = "find different tests delta")
+    parser.add_argument("-x", "--xml-file", default = None,
+            action = "append", help = "xml file")
```

Comment:
add more help here, for example the fact that you can use this argument multiple times.

Reply:
also, add an option for the output, do not hardcode where the data is written.